services/controllerhost/consumer.go

```diff
@@ -397,7 +397,7 @@ func selectNextExtentsToConsume(
 	dstDesc *shared.DestinationDescription,
 	cgDesc *shared.ConsumerGroupDescription,
 	cgExtents *cgExtentsByCategory,
-	m3Scope int) ([]*m.DestinationExtent, int, error) {
```

Comment:
how about leaving this as is ? the additional result param helps with unit testing.